src/CBCNodeListViewController.h

```diff
@@ -83,6 +80,13 @@ FOUNDATION_EXTERN CBCNode *_Nonnull CBCCreatePresentableNavigationTree(void);
  */
 @property(nonatomic, strong, nullable) CBCNode *debugLeaf;
 
+/**
+ This NSDictionary holds all the metadata related to this CBCNode.
+ If it is an example noe, a primary demo, related info,
+ if presentable in Catalog, etc.
+ */
+@property(nonatomic, strong, nonnull) NSDictionary *metadata;
```

Comment:
Please create string constants for any keys that CatalogByConvention itself will use. Unless you have another preferred style, I'd follow Apple's [NSAttributedString convention for keys](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/AttributedStrings/Articles/standardAttributes.html#//apple_ref/doc/uid/TP40004903-SW2).

Document these "standard" keys here and change examples to use them instead of raw strings like `breadcrumbs`.